n26/cli.py

```diff
@@ -360,6 +354,29 @@ def statistics(param_from: int, to: int):
     click.echo(text.strip())
 
 
+def _parse_from_to_timestamps(param_from: datetime or None, param_to: datetime or None) -> Tuple[int, int]:
+    """
+    Parses cli datetime inputs for "from" and "to" parameters
+    :param param_from: "from" input
+    :param param_to: "to" input
+    :return: timestamps ready to be used by the api
+    """
+    from_timestamp = None
+    to_timestamp = None
+    if param_from is not None:
+        from_timestamp = int(param_from.timestamp() * 1000)
+        if param_to is None:
+            # if --from is set, --to must also be set
+            param_to = datetime.utcnow()
+    if param_to is not None:
+        if param_from is None:
+            # if --to is set, --from must also be set
+            from_timestamp = 1
```

Comment:
I'm using `1` here since `0` acted like `None`.
That is the reason I added the `> 0` thingy to the API documentation.